components/RunpkgIcon.js

```diff
@@ -1,70 +1,52 @@
 import { html } from '../utils/rplus.js';
 
 export default html`
-  <svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 380 377">
-    <g filter="url(#filter0_d)">
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="600"
```

Comment:
This fixed width and height breaks the layout currently (pushes the instructional text to the bottom of the page). I think if you remove it and add check the spacing between the paragraph below it.. then we are good to go!

![image](https://user-images.githubusercontent.com/1457604/77163951-202b1b00-6aa7-11ea-853a-30f35c4e9241.png)